test/e2e/bmc.go

```diff
@@ -22,7 +22,7 @@ type BMC struct {
 	// The IP address of the node
 	// Optional. Only needed if e2eConfig variable
 	// SSH_CHECK_PROVISIONED is true
-	IpAddress string `yaml:"ipAddress,omitempty"`
+	IPAddress string `yaml:"ipAddress,omitempty"`
```

Comment:
Should this be discussed from API perspective also?

Reply:
since this is a test, I didnt bother, this wont change BMO API

Reply:
I meant in general changing ipAddress on the API to IPAddress

Reply:
I didn't get you , this type is defined here in test and the changes affected for this are only in test/e2e/ files

Reply:
This is fine, we have no API guarantees for the e2e module (yet). At some point I expect that we would use this in CAPM3 and then we need to be more careful.